<?xml version="1.0"?>
<launch>

  <arg name="robot_name" default="centaur_ros"/>
  <group ns="$(arg robot_name)">
    <node name="rqt_joint_trajectory_controller" pkg="rqt_joint_trajectory_controller" type="rqt_joint_trajectory_controller"/>
  </group>
</launch>


